www/annotationdb/annotationdb.css

```diff
@@ -0,0 +1,443 @@
+div.wy-nav-content{
```

Comment:
Is this essential? It's changing the left margin position compared to the rest of the pages in the site.